### ISSUE TITLE
Download show listing failed

### ISSUE THREAD

**Issue by AlecHex** [tvrenamer/tvrenamer#390]

**Comment by lcba813** [comment]
I have the same problem, first it started with one or two shows with this error, now I am getting errors on all the renamings.

The file names are:
Arrow.S07E21.HDTV.x264-SVA.mkv
DCs.Legends.of.Tomorrow.S04E14.HDTV.x264-SVA.mkv
the.100.s06e02.web.h264-tbs.mkv
the.flash.2014.s05e21.web.h264-tbs.mkv

This is the output for the terminal...

2019/05/08 13:35:16,299 WARN Show#addEpisodeToSeason two episodes found for show DC's Legends of Tomorrow, season 0, episode 0: "Nip | Stuck" (7119615) and "Terms of Service" (7119617)
2019/05/08 13:35:16,314 WARN Show#addEpisodeToSeason two episodes found for show The 100, season 0, episode 0: "Sanctum" (6785212) and "Red Sun Rising" (6792119)
2019/05/08 13:35:16,328 SEVERE FileEpisode#listingsComplete Season #4, Episode #14 not found for show 'DCs.Legends.of.Tomorrow.'
2019/05/08 13:35:16,329 SEVERE FileEpisode#listingsComplete Season #6, Episode #2 not found for show 'the.100.'
2019/05/08 13:35:16,868 WARN Show#addEpisodeToSeason two episodes found for show The Flash (2014), season 0, episode 0: "Gone Rogue" (7131687) and "Snow Pack" (7121487)
2019/05/08 13:35:16,870 SEVERE FileEpisode#listingsComplete Season #5, Episode #21 not found for show 'the.flash.2014.'
2019/05/08 13:35:17,905 WARN Show#addEpisodeToSeason two episodes found for show Arrow, season 0, episode 0: "Lost Canary" (7103512) and "Spartan" (7103515)
2019/05/08 13:35:17,906 WARN Show#addEpisodeToSeason two episodes found for show Arrow, season 0, episode 0: "Confessions" (7103517) and "Brothers & Sisters" (7042042)
2019/05/08 13:35:17,907 SEVERE FileEpisode#listingsComplete Season #7, Episode #21 not found for show 'Arrow.'

**Comment by lcba813** [comment]
I downloaded the latest master branch and compiled it, the problem appears to be solved.  I guess it's time for a new release?

**Comment by AlecHex** [comment]
> 
> 
> I downloaded the latest master branch and compiled it, the problem appears to be solved. I guess it's time for a new release?
If i download the latest one from their site, will it be working then?

**Comment by lairdmacdonald** [comment]
No as the compiled version is still 0.8 which has been out for some time now. Was hoping we would see a new compiled version early November (or even a compiled beta version) from the comments on another thread but nothing yet. :(

**Comment by itsmesarah94** [comment]
I have the same issue. Can somebody share a compiled version with the latest edits? Because a new release doesn't seem very likely at this point.

**Comment by StephKoenig** [comment]
I agree... at the risk of cross posting
I'd love a release that encapsulates the developments since '17. It's such an awesome program that it seems a waste to have the efforts sat unused (and unappreciated!)

**Comment by TwistedFaith84** [comment]
I'm having almost the exact same issue except if I batch one show at a time (with more then 2 episodes); It tends to get the "Download show listings failed" 99.9% of the time (if it's going to fail) but sometimes it actually does work and gets the information and pulls  the info for a few of the episodes.

**Comment by StephKoenig** [comment]
Have you tried v0.8b3?
https://github.com/tvrenamer/tvrenamer/releases

**Comment by TwistedFaith84** [comment]
Yeah I just tried it. and Although it does work a little better its still not 100%. What this program need is something like what theRenamer used where you can manually enter the TVDB ID Code and have the show be scanned in that way. Just a suggestion.

Unfortunately theRenamer is no longer updated and/or working (at least not on windows 10 home) cause that was one of (if not) The Best programs for renaming movies and tv shows.

**Comment by randomname42** [comment]
I've tried v0.8b3 and still seem to be instantly getting Download show listings failed 100% of the time no matter what.

I don't know if recently updating Windows 10 to version 1903 is what broke it?

Could someone please suggest an alternative tv renamer that lets you just drag and drop files/folders to rename? I used to use a different one (afaik therenamer) which did that but that stopped working too when Windows updated something.

All I want is a renamer which lets you drag and drop without having to "set up" each show first. Unfortunately I don't know of any which do that and still work.

**Comment by TwistedFaith84** [comment]
It is partly cause the program still has errors but it's also mostly because TVDB just changed their API again (and I think made it so old API versions no longer work)

**Comment by StephKoenig** [comment]
Also happened to me... running solid until I tried some shows this morning.

Shame becuase it seems like this might be the end of the line for this amazing tool unless one of the forks does an update:

https://techgaun.github.io/active-forks/index.html#tvrenamer/tvrenamer

**Comment by vipuldelwadia** [comment]
Hi, please try the new beta 1.0b4 and let me know if it works for you: https://github.com/tvrenamer/tvrenamer/releases/tag/v1.0b4

**Comment by randomname42** [comment]
Just threw a half dozen shows at it and it works great! No problems at all, except for not finding one show because of not having the year in the filename. (There's more than one The Flash series, and although The Flash 2014 was in the folder name, the year wasn't in the filename so it didn't find the show correctly.)

But other than not being able to find a show via the folder name in a situation like that, seems to be working perfectly.

**Comment by randomname42** [comment]
Now I'm errors like "Unable to find show information for (episode)" and "Could not get episode for show <(show name) / (6 digit number)>".

More episodes now are not working than are working. Some shows no episodes work, including ones that were working a couple days ago.

I don't know if it's tvdb just being temporarily screwy again, or if they changed something that broke it. It was working perfectly a couple days ago when I posted.

**Comment by StephKoenig** [comment]
I found that with a few shows both before and after. I fixed most issues by removing or adding a year  or comma in the appropriate place so it's a closer match for TVDB and it doesn't get confused with show names.

As for episodes it's never really had a problem but other tools have issues with s1x4 os 03x05 style season episode matching...

Not sure if that will help you. Seem to be working well my end so far. Hopefully it's TVDB ironing out the api as I'm sure a lot of people will be hitting it now that they deprecated thier old one.

**Comment by vipuldelwadia** [comment]
> Now I'm getting errors like "Unable to find show information for (episode)" and "Could not get episode for show <(show name) / (6 digit number)>".

Are you able to provide the filenames for the files you're trying to rename? I haven't experienced this myself so without specific examples I won't be able to figure out what's wrong :/

**Comment by randomname42** [comment]
Sorry for the slow reply.

(fails for whole season for these series)
the.100.s06e01.mkv
game.of.thrones.s08e01.mkv
Young.Justice.S03E01.mkv
Avengers Assemble S05E01.mkv

(works for first episodes of season, fails from this episode on, until end of season)
Avengers Assemble S01E08.mp4
Avengers.Assemble.2x06.mp4
Avengers.Assemble.3x06.mkv


Something inconsistent (which might be tvdb's fault) but is screwy:

(fails whole season)
Avengers.Assemble.4x01.mkv

(succeeds whole season)
Avengers Assemble Secret Wars S04E01.mkv

Secret Wars is the season 4's name/arc.

**Comment by vipuldelwadia** [comment]
This is the TheTVDB v1 API shutdown. Releases before 1.0b5 looked up episodes there, and that API is retired, so those versions cannot find any show information at all.

1.0 uses TVmaze instead: https://github.com/tvrenamer/tvrenamer/releases/tag/v1.0

Closing. If you still see this on 1.0, please reopen or open a new issue with an example filename.
